lib/appsignal/phoenix.ex

```diff
@@ -46,7 +46,9 @@ if Appsignal.phoenix? do
       # explicitly remove the transaction here so the regular error handler doesn't submit it again
       :ok = TransactionRegistry.remove_transaction(transaction)
 
-      Logger.debug("Submitting Phoenix error #{inspect transaction}: #{message}")
+      Logger.debug(fn() ->
+        "Submitting Phoenix error #{inspect transaction}: #{message}"
+      end)
```

Comment:
Not sure I understand this one. What does this fix by passing a function to the logger? I can't get dialyzer to list a problem if I undo this change

Reply:
Sorry, this one slipped in and is a fix for a Credo issue, not for Dialyzer. For `Logger.debug/1` calls with interpolation, passing a function with no argument is [preferred](https://github.com/rrrene/credo/issues/299). Not an issue in our case, but since it's enforced by Credo, we might as well follow its advice.